simulation/sub8_montecarlo/montecarlo_tests/montecarlo_pd_controller.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+PKG = 'sub8_montecarlo'
+NAME = 'test_controller'
+
+import sys
+import unittest
+import rospy
+import rostest
+from sub8_montecarlo_tools import VerifyController
+
+
+class TestController(unittest.TestCase):
+    def setUp(self, *args):
+        '''TODO:
+            - Assert that wrenches within bounds are close to the unbounded least-squares estimate
+            - Make waiting for node functionality into a test_helpers util
+            - Test random wrenches and assert valid behavior
+        '''
+        self.num_runs = rospy.get_param('runs', 4)
+        self.time_limit = rospy.get_param('run_time', 10)
+
+    def test_pd_controller(self):
```

Comment:
You'll probably want to rename this unit test at some point to make the name more accurately represent what you're testing here. This is especially the case if you're going to be adding more unit tests for the controller.